frontend/src/views/Create.tsx

```diff
@@ -21,7 +25,9 @@ function CreateGamePage() {
     };
     createRoom(roomHost)
       .then((res) => {
-        redirectToLobby(res, roomHost.host);
+        dispatch(setRoom(res));
+        dispatch(setCurrentUser(res.host));
+        redirectToLobby(res, res.host);
```

Comment:
Yeah for now I kept that since otherwise it won't persist across refreshes, but in the future we could definitely look into things like local storage or redux persist to simplify that process